src/utils.ts

```diff
@@ -10,15 +10,27 @@ import { config } from "./config";
 const isWin = /^win/.test(process.platform);
 const dartExecutableName = isWin ? "dart.exe" : "dart";
 const pubExecutableName = isWin ? "pub.bat" : "pub";
+const flutterExecutableName = isWin ? "flutter.bat" : "flutter";
 export const dartVMPath = "bin/" + dartExecutableName;
 export const dartPubPath = "bin/" + pubExecutableName;
 export const analyzerPath = "bin/snapshots/analysis_server.dart.snapshot";
 export const extensionVersion = getExtensionVersion();
 export const isDevelopment = checkIsDevelopment();
 
+export function isFlutterProject(): boolean {
+	if (workspace.rootPath)
+		if (fs.existsSync(path.join(workspace.rootPath, "pubspec.yaml")))
+			return fs.readFileSync((path.join(workspace.rootPath, "pubspec.yaml"))).includes("sdk: flutter");
```

Comment:
I think it would be slightly better to use a regex here to be flexible about the whitespace between `sdk:` and `flutter` (maybe even allow option whitespace before the colon; not sure how struct YAML is).